ui/stories/sankey.stories.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { storiesOf } from '../.storybook/facade';
+import Sankey from '../src/component/sankey';
+
+const source = [
+  ['key', 'a', 'b', 'c', 'd', 'e', 'f'],
```

Comment:
To me , below data structure makes more sense, it demonstrates useful meaning of the data. With below data structure, the data processing in chart component will also be easy.

```
[Source, Target, Value]
['a', 'b', 2]
['c', 'd', e'']
...
```

Reply:
Think about the data format we defined before:
```js
export const source = [
  ['product', '2015', '2016', '2017'],
  ['Matcha Latte', 43.3, 85.8, 93.7],
  ['Milk Tea', 83.1, 73.4, 55.1],
  ['Cheese Cocoa', 86.4, 65.2, 82.5],
  ['Walnut Brownie', 72.4, 53.9, 39.1],
];
```
From this definition we have:
1. The first row indicates dimensions(y axis) 
2. The first column indicates categories(x axis).
3. The values are defined as a 2-dim matrix.

So in order to uniform the data format, it is better to follow this data structure and do not create a special format for this type of chart.